ng-dev/pr/merge/BUILD.bazel

```diff
@@ -34,27 +34,18 @@ ts_library(
     srcs = glob(["**/*.spec.ts"]),
     deps = [
         ":merge",
-        "//dev-infra/release/config",
-        "//dev-infra/release/versioning",
-        "//dev-infra/utils",
-        "//dev-infra/utils/testing",
-        "@npm//@types/jasmine",
-        "@npm//@types/node",
+        "//ng-dev/release/config",
+        "//ng-dev/release/versioning",
+        "//ng-dev/utils",
+        "//ng-dev/utils/testing",
         "@npm//@types/node-fetch",
         "@npm//nock",
     ],
 )
 
 jasmine_node_test(
     name = "test",
-    # Disable the Bazel patched module resolution. It always loads ".mjs" files first. This
-    # breaks NodeJS execution for "node-fetch" as it uses experimental modules which are not
-    # enabled in NodeJS. TODO: Remove this with rules_nodejs 3.x where patching is optional.
-    # https://github.com/bazelbuild/rules_nodejs/commit/7d070ffadf9c3b41711382a4737b995f987c14fa.
-    args = ["--nobazel_patch_module_resolver"],
-    deps = [
+    specs = [
         ":test_lib",
-        "@npm//node-fetch",
-        "@npm//semver",
```

Comment:
Out of curiosity, do you know why these deps can be removed now? do the tests still pass?

Reply:
Fetch and semver aren't actually used in these tests, so they were being provided needlessly

Reply:
👍